client/my-sites/checkout/checkout-thank-you/hundred-year-plan-thank-you/index.tsx

```diff
@@ -0,0 +1,172 @@
+	border: 1px solid var( --gray-gray-0, #f6f7f7 );
+	background-color: var( --studio-black );
+	background: linear-gradient( #c1c0d3, #e3e2f3, #c1c0d3 );
+	color: var( --studio-black );
+`;
+
+const CustomizedWordPressLogo = styled( WordPressLogo )`
+	margin: 0;
+	fill: var( --studio-white );
+`;
+
+export default function HundredYearPlanThankYou( { siteSlug, receiptId }: Props ) {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const isUserJoinedWithinADay = useSelector( ( state ) =>
+		isUserRegistrationDaysWithinRange( state, null, 0, 1 )
+	);
+	const receipt = useSelector( ( state ) => getReceiptById( state, receiptId ) );
+	const isReceiptLoading = ! receipt.hasLoadedFromServer || receipt.isRequesting;
+	useEffect( () => {
+		dispatch( hideMasterbar() );
+		if ( isReceiptLoading && receiptId ) {
+			dispatch( fetchReceipt( receiptId ) );
+		}
+	}, [ dispatch, isReceiptLoading, receiptId ] );
+
+	if ( ! isReceiptLoading && ! receipt.data?.purchases?.length ) {
+		page( '/' );
+	}
+	const isMobile = useMobileBreakpoint();
+	return (
+		<>
+			<Global
+				styles={ css`
+					body.is-section-checkout,
+					body.is-section-checkout .layout__content {
+						background: linear-gradient( 233deg, #06101c 2.17%, #050c16 41.26%, #02080f 88.44% );
+					}
+				` }
+			/>
+			<MasterBar>
+				<div>
+					{ ' ' }
+					<CustomizedWordPressLogo size={ 24 } />
+				</div>
+			</MasterBar>
+			<PageContainer>
+				{ ! isReceiptLoading && (
+					<Content isMobile={ isMobile }>
+						<div className="hundred-year-plan-thank-you__thank-you-text-container">
+							<Header className="wp-brand-font" isMobile={ isMobile }>
+								{ translate( 'Your legacy is in safe hands' ) }
+							</Header>
+							<Highlight>
+								<p>
+									{ translate(
+										"Congratulations on securing the 100-Year Plan. We've applied your exclusive, tailor-made benefits to %(siteSlug)s.",
+										{ args: { siteSlug } }
+									) }
+								</p>
+								<p>
+									{ translate(
+										'If you have any questions or need assistance with anything at all, our dedicated Premier Support team are standing by to help.'
+									) }
+								</p>
+							</Highlight>
+							<ButtonBar isMobile={ isMobile }>
+								{ isUserJoinedWithinADay ? (
+									<StyledButton onClick={ () => page( `/plans/my-plan/${ siteSlug }` ) }>
+										{ translate( 'View plan benefits' ) }
+									</StyledButton>
+								) : (
+									<StyledButton
+										onClick={ () => page( `/setup/site-setup/goals?siteSlug=${ siteSlug }` ) }
+									>
+										{ translate( 'Start building' ) }
+									</StyledButton>
+								) }
+								<StyledLightButton onClick={ () => page( '/help' ) }>
+									{ translate( 'Access premium support' ) }
+								</StyledLightButton>
+							</ButtonBar>
+						</div>
+						<video
+							src="https://wpcom.files.wordpress.com/2023/08/century-100-banner.mp4"
+							preload="metadata"
```

Comment:
```suggestion
							preload="auto"
```

Setting this to `auto` will fetch the video instead of just the metadata.